web/frontend/src/components/slam/index.svelte

```diff
@@ -53,18 +61,42 @@ const deleteDestinationMarker = () => {
   destination = undefined;
 };
 
-const refresh2d = async () => {
+const startDurationTimer = (start: number) => {
+  durationInterval = setInterval(() => {
+    sessionDuration = Date.now() - start;
```

Comment:
Small problem that I can see here: this will occasionally cause users to see skipped seconds due to the nature of setTimeout, since it only guarantees minimum time until re-entering the event loop. Athough its a small annoyance I could totally see people reporting this asking if there's a problem. There's an entire [google discussion](https://www.youtube.com/watch?v=MCi6AZMkxcU) on this 😂 . I think in the future it might be a good idea to hook this timer into the requestAnimationFrame loop in the child 2d-renderer component, but since that's a bit complex could we maybe for now just reduce the interval down to 400ms?